src/main/java/hibernate/tournament/model/Tournament.java

```diff
@@ -0,0 +1,29 @@
+package hibernate.tournament.model;
+
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+
+import javax.persistence.*;
+
+
+@Getter
+@Setter
+@Entity
+@Table(name = "tournament_winners")
+@AllArgsConstructor
+@NoArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class Tournament {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    Integer id;
+
+    @Column(name = "team_name")
```

Comment:
Why intentions are so different?
If you want your column called team_name, use in Java the same name.
Or vice versa.